media/src/utils.js

```diff
@@ -27,11 +27,31 @@ const joinUrl = function(base, path) {
     return url.href;
 };
 
+/**
+ * Converts parameters and objects into URL Search Parameters
+ * @param {*} params 
+ * @param {*} objects 
+ * @returns URLSearchParams
+ */
+const passObjects = function(params, objects) {
+    objects.forEach((object, index) => {
+        const prefix = `obj${index}_`;
+        params[prefix + "kind"] = object.kind;
+        if (object.params) {
+            for (const [key, value] of Object.entries(object.params)) {
+                params[prefix + "params_" + key] = value;
+            }
+        }
+    });
+    const urlParams = new URLSearchParams(params);
+    return urlParams;
+}
+
 /**
  * Given a room/session id, return its path.
  */
-const getRoomUrl = function(id) {
-    return joinUrl(window.location.origin, `/rooms/${id}/`);
+const getRoomUrl = function(id, objects) {
```

Comment:
If we go with this method, `objects` would need to be an optional argument here, otherwise we get this error on join room:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'forEach')
    at passObjects (bundle.js:126771:17)
    at getRoomUrl (bundle.js:126788:65)
    at onJoinRoom (bundle.js:163865:30)
    at HTMLFormElement.submit_handler (bundle.js:163928:35)
    at HTMLFormElement.<anonymous> (bundle.js:207:23)
```

And making a room with no objects in it shouldn't append a `?` to the url path.